pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -602,12 +615,19 @@ def _get_protocol_row(self, protocol):
 
         return None
 
+    @property
+    def cdn_used_as_installation_source(self):
```

Comment:
Please, use `payload.get_source_type == SOURCE_TYPE_CDN` instead of this property and the function below.

Reply:
Changed.